HOWTO.md

```diff
@@ -576,6 +601,25 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
 
 ## How to run the Client
 
+### `fdo-manufacturing-client`
+
+1. If you come across use case such as: you need to check DI (Device Initialize)
+or DIUN (Device Initialize over Untrusted Network) protocol is working as expected, 
+then you can run `fdo-manufacturing-server` and `fdo-manufacturing-client`
+on their own. 
+2. Run `fdo-manufacturing-server` as mentioned in [Manufacturing Server](#manufacturing-server). 
+3. If you are running `plain_DI`, set following environmental variables additionally
+ before running manufacturing client:
+    - `DI_MFG_STRING_TYPE`, `DI_KEY_STORAGE_TYPE`, `DI_SIGN_KEY_PATH`, `DI_HMAC_KEY_PATH`
+Check section [Environmental Variables](#environmental-variables) for detail of each env variable.
+4. If you are running `DIUN` use below command (above step 3 is not needed): 
+   ```
+   sudo MANUFACTURING_SERVER_URL=http://localhos:8080 DEVICE_CREDENTIAL=/path/to/device-credentials 
+   DIUN_PUB_KEY_ROOTCERTS=/path/to/keys/diun_cert.pem ./path/to/binary/fdo-manufacturing-client
+   ```
+    
```

Comment:
You are missing a first step telling where the Device Credentials are being looked into by default.